### PR TITLE
Feature/unit test endpoint cx 4596

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Currently, we have no test coverage for the function that uploads snapshot and `live_photo` to the Onfido Api.

# Solution

- Added unit tests for `sendMultiframeSelfie` function
- Split the `Confirm` component into different files

Please start your review [here](https://github.com/onfido/onfido-sdk-ui/pull/902/files#diff-2622f057e7289156da7a341309a43842R1), the rest the code has just been copy-pasted in different files

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -141,6 +141,7 @@
     "blueimp-load-image": "~2.12.2",
     "classnames": "~2.2.5",
     "enumerate-devices": "^1.1.0",
+    "eslint-plugin-jest": "^23.6.0",
```
Comment (josokinas) [inline package.json]: I think this should go to devDependencies.

File: src/components/utils/unit_tests/onfidoApi.test.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { sendMultiframeSelfie } from "../onfidoApi"
+
+let jwtToken = 'my_token'
```
Comment (josokinas) [inline src/components/utils/unit_tests/onfidoApi.test.js]: You don't need to override this, so `const` would be more appropriate.

File: package.json (hunk 1 above)
Comment (stefaniacardenas) [reply package.json]: Oh, I thought I had run it with `--save-dev`. Good spot!

File: src/components/utils/unit_tests/onfidoApi.test.js (hunk 2 above)
Comment (stefaniacardenas) [reply src/components/utils/unit_tests/onfidoApi.test.js]: True. I was overriding in a test that I canceled and I forgot to change it back. Thanks

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://902-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/utils/onfidoApi.js [hunk 3]
```diff
@@ -46,8 +46,28 @@ export const sendMultiframeSelfie = (snapshot, selfie, token, url, onSuccess, on
     const snapshot_uuids = JSON.stringify([res.uuid])
     uploadLivePhoto({ file: { blob, filename }, sdkMetadata, snapshot_uuids}, url, token, onSuccess, onError)
   })
-  .catch((res) => {
-    onError(res)
+  .catch(() => {
+    // TODO when the backened enpoint will be ready please uncomment this line
+    // onError(res)
+
+    // TODO when the backened enpoint will be ready please delete this code
```
Comment (kopijunkie) [inline src/components/utils/onfidoApi.js]: Minor: typos (lines 50, 53, 63) - 1 too many "e" in "backend" & missing "d" in "endpoint" ;)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-902-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://902-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**